tools/report-converter/tests/unit/analyzers/tidy_output_test_files/tidy3_cpp.plist

```diff
@@ -22,38 +22,37 @@
 				<key>line</key>
 				<integer>4</integer>
 			</dict>
-			<key>path</key>
+			<key>notes</key>
```

Comment:
Is this hard-coded to only accept this value?

Creating a new "note kind" that is "FixIt" and rendering it separately (e.g., with a green backgrounded box or something?) would make it stand out more. FixIts are technically more than notes.

Reply:
Extracted to #4064 instead. The other comment with the rebase still stands.